### PR TITLE
reference messages.default in ScalaI18N.md

### PR DESCRIPTION
**Pull request by fnmeissner**
In messages.default the default messages for forms validation are defined. 
In order to overwrite the validation messages with language-specific messages, you need to know under which alias the messages are defined.
The reference might save users some time searching for the aliases that play uses.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @cap3fme,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by benmccann** [comment]
@cap3fme the link doesn't work:
>[error] Bad relative link ../../../../../../framework/src/play/src/main/resources/messages.default at /home/travis/build/playframework/playframework/documentation/manual/working/scalaGuide/main/i18n/ScalaI18N.md:56
[error] To see which messages can be overwritten, you can have a look at [`messages.default`](../../../../../../framework/src/play/src/main/resources/messages.default).
[error]                                                                                                  ^
java.lang.RuntimeException: Documentation validation failed
	at com.typesafe.play.docs.sbtplugin.PlayDocsValidation$$anonfun$16.apply(PlayDocsValidation.scala:441)
	at com.typesafe.play.docs.sbtplugin.PlayDocsValidation$$anonfun$16.apply(PlayDocsValidation.scala:315)

**Comment by cap3fme** [comment]
Thanks for your notice.
I have changed the url to an absolute path pointing at the source in Github. 
This is not an optimal solution, but I am currently not aware of any better solution. Since the file lies inside the resources-folder, I think it is not possible to use a reference to the api-documentation.
Please let me know if there is anything I can do and feel free to reject my proposal if it does not meet the documentation requirements.
With kind regards,
Felix

File: documentation/manual/working/scalaGuide/main/i18n/ScalaI18N.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ class MyService @Inject()(langs: Langs, messagesApi: MessagesApi) {
 }
 ```
 
+Play provides predefined messages for forms validation. 
+You can overwrite these messages either with the default message file or any language-specific message file.
+To see which messages can be overwritten, you can have a look at [`messages.default`](https://github.com/playframework/playframework/blob/2.5.x/framework/src/play/src/main/resources/messages.default)(you need to select the branch according to the version of play you are using).
```
Comment (gmethvin) [inline documentation/manual/working/scalaGuide/main/i18n/ScalaI18N.md]: I think you can use %PLAY_VERSION% here instead of the version.

**Comment by marcospereira** [comment]
Hey @cap3fme, are you still interested in this change? I think that after the suggestion @gmethvin made, we can merge this. :-)

What do you think?

**Comment by PromanSEW** [comment]
@cap3fme please update PR for merging

**Comment by cap3fme** [comment]
The replacement of %PLAY_VERSION% does not happen when used inside links, so this is not going to work. Any idea how I could fix this?

**Comment by richdougherty** [comment]
OK I had a look into this. The documentation is rendered dynamically on playframework.com by play-doc. Here's the code which hooks into the pegdown parsing/rendering pipeline:

https://github.com/playframework/play-doc/blob/20a1ca574ade9cbfb69dd199cdff002dc19bd471/src/main/scala/play/doc/PlayDoc.scala#L199-L214

It shouldn't be too hard to get some sort of link rewriting scheme into there, but it would take some research to investigate how pegdown works, rebuild play-doc, etc.

Another option might be for us to copy the messages files during the build so that they get packaged into the documentation archive. If they were packaged then it would be possible to use a relative link.

**Comment by marcospereira** [comment]
Fixed this by adding `.default` files to play-docs bundle. The content then can be embedded like we do for some `reference.conf` files. See the image below:

screen shot 2019-01-04 at 5 42 08 pm

**Comment by wsargent** [comment]
@marcospereira good to merge?

**Comment by lightbend-cla-validator** [comment]
Hi @mergify[bot],

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by lightbend-cla-validator** [comment]
Hi @mergify[bot],

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Pull request merged**